pylint/extensions/docparams.py

```diff
@@ -222,8 +223,11 @@ def visit_functiondef(self, node: nodes.FunctionDef) -> None:
         # skip functions smaller than 'docstring-min-length'
         lines = checker_utils.get_node_last_lineno(node) - node.lineno
         max_lines = get_global_option(self, "docstring-min-length")
-        if max_lines > -1 and lines < max_lines:
-            return
+        if isinstance(max_lines, int):
```

Comment:
This is a similar issue to the `assert` I have just removed (due to `linter.stats` having multiple types). Is this okay? Or do you want me to add `type: ignore` here as well?

Reply:
I think this could be checked once when parsing the option, so we can also ignore.

Reply:
Done!